CONTRIBUTING.md

```diff
@@ -4,9 +4,9 @@ In order to contribute to `nf-synapse`, you will first need to either create a f
 
 ## Create New Modules
```

Comment:
Is this a good place to talk about how you can leverage `bin/` scripts in your modules?

This would be a good example to provide: https://github.com/Sage-Bionetworks-Workflows/nf-synapse/blob/main/modules/get_user_id.nf